packages/dd-trace/test/opentracing/propagation/text_map.spec.js

```diff
@@ -442,6 +443,7 @@ describe('TextMapPropagator', () => {
       }
       const spanContextA = propagator.extract(carrierA)
       expect(spanContextA._baggageItems).to.deep.equal({})
+      expect(getAllBaggageItems()).to.be.undefined
```

Comment:
Is it expected that this can return undefined? As a user I would expect to get an empty object when there are no items. We don't necessarily need to actually store an empty object underneath but I think we should return one when the store is empty.

Reply:
I asked for clarification in the RFC